### DISCUSSION TITLE
Enhanced comments on track

### DISCUSSION THREAD

**Discussion by obsidienne** [puemos/radiopush#19] [Ideas]
My users want to share their thought on the songs (is it easy to dance, more a romantic song for the end of the night...) and videos. 

We could add under each posts a comment. 

What do you think ?

**Comment by puemos** [comment]
@obsidienne so I had this idea in the beginning but I dropped it because it will require in some point moderation efforts.
My counter proposal is to have a set of tags that we can create that people can use on songs 
What do you think?

**Reply by obsidienne** [reply]
Sure, it will need moderation.
In my case, the goal is to post videos on song, like dancing video. The goal behind to show how this specific track can be danced, with a link to YouTube exclusively.

**Reply by puemos** [reply]
Got it, so we need to double check the terms of service and add a "report" functionality

**Reply by obsidienne** [reply]
Report functionality ? To report an infrigement ?

**Reply by puemos** [reply]
Exactly

**Comment by obsidienne** [comment]
I will fix the other issue before tackling this one. 
Need more reflexion on the need before starting coding

**Comment by puemos** [comment]
Let's move this discussion to the new repo
